.beetbox/Vagrantfile

```diff
@@ -29,12 +31,19 @@ vconfig = {
   'drush_create_alias' => true
 }
 
-# Create default config file.
-if !File.exist?(project_config)
-  FileUtils.mkdir_p config_dir
-  open(project_config, "w+") << "---\nbeet_domain: #{vconfig['beet_domain']}\n"
+# Create config directory.
+FileUtils.mkdir_p config_dir
+
+# Create config.yml from composer config.
+if File.exist?(composer_json)
+  composer_conf = JSON.parse(File.read(composer_json))
+  cconfig = composer_conf['extra']['beetbox']['config'] rescue nil
```

Comment:
can we have a message in case `composer_conf` is not in proper JSON format.

Also, `rescue nil` is not a good practice. better to use 
```
cconfig = composer_conf['extra']['beetbox']['config'] rescue nil
if cconfig
  open(project_config, "w+") << cconfig.to_yaml if cconfig.is_a?(Hash)
```